### ISSUE TITLE
Plugins need their own page in docs

### ISSUE THREAD

**Issue by evilnick** [juju/docs#1246]
There are a number of plugins for Juju, some of which are essential to certain operations. Whilst these are explored in relevant sections of the docs, there are a number of things common to plugins (e.g. #1191) which would benefit from being in a central location. Many people don't even know they have/are using plugins!

**Comment by pmatulis** [comment]
This below situation can probably be improved too:

```
$ grep -i 'charm-tools' * | grep apt
authors-charm-building.md:sudo apt-get install charm-tools
charms-deploying.md:sudo apt install charm-tools
developer-getting-started.md:sudo apt-get install charm-tools
tools-charm-tools.md:sudo apt-get update && sudo apt-get install charm-tools

**Comment by evilnick** [comment]
Charm tools isn't a plugin, but yes, they should all refer to the tools page for installing. I will generate another issue.

**Comment by evilnick** [comment]
this issue was closed with #1252
